src/commands.cpp

```diff
@@ -2735,7 +2753,7 @@ bool CommandRemoveContact::procresult(Result r)
 
         if (User *u = client->finduser(email.c_str()))
         {
-            u->show = v;
+            u->show = HIDDEN;
```

Comment:
We fixed this issue in the refactoring branch. It should be `v`, because it may eventually become `BLOCKED`, in example. I'll re-adjust it ;)

Reply:
Done